sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/xml/HomeFragment.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.sample.compose.ui.screens.xml
```

Comment:
Shall we remove everything which is related to `xml`? 

Same goes for `HomeViewModel.kt`